tutorials/gitlab-instant-app/index.mdx

```diff
@@ -22,43 +22,68 @@ This page shows you how to use the GitLab repository manager InstantApp on your
   - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
 </Message>
 
-GitLab is a web-based Git repository manager with wiki and issue tracking features. GitLab includes git repository management, code reviews, an issue tracking, wikis and much more.
+GitLab serves as a web-based manager for Git repositories, offering features like wikis and issue tracking. It provides a myriad of functionalities, including management of git repositories, code reviews, issue tracking, and wikis, among others.
```

Comment:
I am under the impression that these two sentences repeat themselves a bit, don't they?

Reply:
Agreed.